sdk/common/src/main/java/io/opentelemetry/sdk/common/export/RetryPolicyBuilder.java

```diff
@@ -3,34 +3,36 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.exporter.internal.retry;
+package io.opentelemetry.sdk.common.export;
 
 import static io.opentelemetry.api.internal.Utils.checkArgument;
 import static java.util.Objects.requireNonNull;
 
 import java.time.Duration;
 
-/**
- * This class is internal and is hence not for public use. Its APIs are unstable and can change at
- * any time.
- */
+/** Builder for {@link RetryPolicy}. */
 public final class RetryPolicyBuilder {
```

Comment:
Can you use https://github.com/google/auto/blob/main/value/userguide/builders-howto.md#autobuilder?

Reply:
👍